test/tests/kernels/fv_adapt/transient-adapt.i

```diff
@@ -0,0 +1,116 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+  nz = 0
+  zmax = 0
+  elem_type = QUAD4
+[]
+
+[Variables]
+  [./u]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [v][]
```

Comment:
`u `and `v` can use the same `[../]` or` []`?

Reply:
fixed